__tests__/app.test.js

```diff
@@ -121,6 +123,25 @@ describe('/api/articles', () => {
             })
         })
     })
+    test('GET 200: Should return an array of articles that meet the provided query of topic', () => {
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({ body: { articles }}) => {
+            expect(articles.length).toBe(1);
+            articles.forEach((article) => {
+                expect(article.topic).toBe('cats')
+            })
+        })
+    })
```

Comment:
You should add one more 200 test to check that if you do a query for a topic that exists but has no articles, it will return an empty array 🔍